packages/tools/src/settings.ts

```diff
@@ -0,0 +1,22 @@
+class GlobalToolSettings {
```

Comment:
how about setting these options per tool?
as a general pattern we could use a factory function to create a tool, e.g. `wiki()` instead of `wiki` - then we can pass parameters to create a tool, e.g.:
```
documentGenerator({ outputDir })
```